src/comments.js

```diff
@@ -381,6 +383,20 @@ function handleConditionalExpressionComments(enclosingNode, followingNode, comme
   return false;
 }
 
+function handleAssignmentPatternComments(enclosingNode, comment) {
```

Comment:
Oh, and I forgot to mention that in this case there is a trailing comment (I can _trace_ it here https://github.com/jlongster/prettier/blob/master/src/comments.js#L193).